src/index.tsx

```diff
@@ -2,20 +2,8 @@ import 'bootstrap';
 import React from 'react';
 import { render } from 'react-dom';
 
-import { UserAction } from './common/services/user';
-import { SpinnerActions } from './packages/spinner/ducks';
-
-import 'app/styles/main.css';
-
 import Root from 'app/Root';
 
-type RootAction = UserAction | SpinnerActions; // TODO add router actions
-
-// TODO can be in some 'types.d.ts' & react-app-env.d.ts doesn't work, keeping here now
-declare module 'typesafe-actions' {
-  interface Types {
-    RootAction: RootAction;
-  }
-}
+import 'app/styles/main.css';
```

Comment:
style must be above importing Root because once you import Root, we start rendering of the App but without loaded styles

Reply:
to sa mi nezda, ved webpack predna nezameni ten import za style tag? ako to funguje?

som prec, tak mozes to prehodit a daj tam pls komentar aby to niekto nepresunul.